amazon/amazon_test.go

```diff
@@ -0,0 +1,135 @@
+package amazon
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"golang.org/x/xerrors"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Update(t *testing.T) {
+	testCases := []struct {
+		name          string
+		xmlResponse   string
+		gzipFileName  string
+		expectedError error
+	}{
+		{
+			name:          "happy path",
+			xmlResponse:   goodRepoMDXMLResponse,
+			gzipFileName:  "test-data/updateinfo.xml.gz.valid",
```

Comment:
```suggestion
			gzipFileName:  "testdata/updateinfo.xml.gz.valid",
```
I think `testdata` is popular.
https://github.com/golang/go/tree/master/src/cmd/go/testdata

Reply:
In addition, haven't you pushed `testdata` yet?

Reply:
Yeah sorry I forgot to push that, but I believe you've already added this so I'll resolve.